lib/ws-channel.js

```diff
@@ -56,43 +59,68 @@ WebsocketChannel.prototype._onNotification = function(message) {
 function nullCallback() {
 }
 
-WebsocketChannel.prototype.connect = function(uri, _token) {
-  var token = _token || url.parse(uri).auth;
+WebsocketChannel.prototype.connect = function(uri, options) {
```

Comment:
do we need to change this? I'd hope that the sessionToken would be completely internal. Its nice to have a `getToken()` for debug purposes, but I can't see every calling connect with a session token.

Unless .connect is also called internally on reconnection, but then perhaps it should be split into a _connect.

I'll keep reading to see if that's the case :-)